### PR TITLE
Add Dependabot for GitHub Actions updates

### PR DESCRIPTION
**Pull request by nielsvanvelzen**
**Changes**
- Add Dependabot for GitHub Actions updates
  - Mostly copied from jellyfin-sdk-kotlin (originally added by @h1dden-da3m0n)

**Issues**

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+version: 2
+updates:
+  - package-ecosystem: github-actions
+    directory: /
+    schedule:
+      interval: weekly
+    labels:
+      - ci
+      - github-actions
```
Comment (h1dden-da3m0n) [inline .github/dependabot.yml:9]: I know I am the dumb a$$ who added these, but could you add `- dependencies`?
I should have done so too, and will fix it for the jf-sdk

**Review by h1dden-da3m0n — APPROVED** [review]

File: .github/dependabot.yml (hunk 1 above)
Comment (nielsvanvelzen) [reply .github/dependabot.yml:9]:
```suggestion
      - ci
      - github-actions
      - dependencies
```

**Comment by nielsvanvelzen** [comment]
Note to merger: add the labels or dependabot will fail

**Comment by h1dden-da3m0n** [comment]
> Note to merger: add the labels or dependabot will fail

It wont fail per say, but complain in the created PRs

**Review by thornbill — APPROVED** [review]

**Pull request merged**